### PR TITLE
Add profile limits

### PR DESCRIPTION
**Pull request by ghost**
Tested with adding very long text.
Confirmed with checking validation messages and input.

# Confirmation
- [My GitHub Profile's name and bio](https://github.com/GrpeApple/)
	- Profile name validation message
	- Profile bio validation message
- Profile URL validation message
- Profile Twitter username validation message
- Profile company validation message
- Profile location validation message

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,6 +40,55 @@ Note: longer team names will still validate, they will just be truncated.
...
+*This was verified through checking validation messages and input when attempting to change the profile url.*
+
+#### Profile Twitter usernames
+
+- Max length: 15 characters
+- All characters must be either an underscore (`_`), or alphanumeric
+
+Note: leading/trailing whitespace will still validate, they will just be trimmed.
+
+*This was verified through checking validation messages and input when attempting to change the profile twitter username.*
```
Comment (dead-claudia) [inline README.md]: [Those are all Twitter restrictions, not necessarily GitHub restrictions, and Twitter documents this well.](https://help.twitter.com/en/managing-your-account/twitter-username-rules) Not sure the point of including it here is unless GitHub somehow diverges from Twitter in what it accepts (which would itself be an obvious bug on GitHub's end if it did ever happen).

**Review by dead-claudia — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (ghost) [reply README.md]: Alright

Comment (dead-claudia) [reply README.md]: Could you remove them then?

Comment (ghost) [reply README.md]: Sure

Comment (ghost) [reply README.md]: Should I rebase and force push or use two commits?

Comment (dead-claudia) [reply README.md]: Up to you - I can squash and merge if you don't.

Edit: don't worry about it.

**Review by dead-claudia — APPROVED** [review]

**Comment by ghost** [comment]
Ok

**Pull request merged**